core/domain/exp_services.py

```diff
@@ -1198,13 +1220,13 @@ def regenerate_exploration_and_contributors_summaries(exploration_id):
         exploration_id: str. ID of the exploration.
     """
     exploration = exp_fetchers.get_exploration_by_id(exploration_id)
-    exp_summary = _compute_summary_of_exploration(exploration)
+    exp_summary = compute_summary_of_exploration(exploration)
     exp_summary.contributors_summary = (
         compute_exploration_contributors_summary(exp_summary.id))
     save_exploration_summary(exp_summary)
 
 
-def _compute_summary_of_exploration(exploration):
+def compute_summary_of_exploration(exploration):
```

Comment:
If you want to call this method in Beam it cannot contain `get_by_id`

Reply:
Update to get function.

Reply:
It cannot be `get` either, you all  NDB operations should be done through `GetModels`, `PutModels` or `DeleteModels`